### PR TITLE
ORC-1279: Update the cmake version in ubuntu18 dockerfile to latest

### PR DESCRIPTION
**Pull request by deshanxiao**
### What changes were proposed in this pull request?
Update cmake of ubuntu18 to latest.

### Why are the changes needed?
[add_compile_definitions ](https://cmake.org/cmake/help/latest/command/add_compile_definitions.html)only supprt cmake 3.12+. But the default image of ubuntu18 only support cmake 3.10. We should upgrade it.

### How was this patch tested?
Existing UT

### CODE REVIEW

**Comment by deshanxiao** [comment]
I have checked other OS and only Ubuntu18.04 has the problem.

**Comment by dongjoon-hyun** [comment]
cc @williamhyun , too

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
+          - ubuntu-18.04
```
[the remark this answers is not in the document]
Comment (deshanxiao) [reply .github/workflows/build_and_test.yml]: Sure, thank you~ @dongjoon-hyun

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you for update, @deshanxiao .

**Review by williamhyun — APPROVED** [review]
+1 LGTM

Thank you @deshanxiao !

**Pull request closed without merging**